Gemfile.lock

```diff
@@ -217,7 +224,7 @@ GEM
     libv8 (3.16.14.17)
     loofah (2.0.3)
       nokogiri (>= 1.5.9)
-    mail (2.6.6)
+    mail (2.6.4)
```

Comment:
No they are not necessary. The Gemfile.lock changed automatically and I was not sure if I should upload it.